src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
I still think that the function namespace does not convey anything. Just call it `utils.bind`